lua/nvim-lsp-installer/fs.lua

```diff
@@ -0,0 +1,42 @@
+local uv = vim.loop
```

Comment:
are you sure you're not re-inventing the wheel here?

Reply:
`vim.loop` (libuv) is pretty much the only Lua API for filesystem operations, that I am aware of. And it's very low level, hence this module. Plugins like [plenary.nvim](https://github.com/nvim-lua/plenary.nvim) exists to make it easier to work with.